Description: sudoers: fix path to ovirt-shutdown-wrappers.sh
 .
 ovirt-guest-agent (1.0.9.1-2.1) unstable; urgency=low
 .
   * Non-maintainer upload.
   * sudoers: fix path to ovirt-shutdown-wrappers.sh
Author: Vinzenz Feenstra <evilissimo@redhat.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ovirt-guest-agent-1.0.9.1.orig/scripts/sudoers.ovirt-guest-agent
+++ ovirt-guest-agent-1.0.9.1/scripts/sudoers.ovirt-guest-agent
@@ -1,6 +1,6 @@
 Cmnd_Alias OVIRTAGENT_SCRIPTS =\
     /usr/share/ovirt-guest-agent/ovirt-hibernate-wrapper.sh *,\
-    /usr/share/ovirt-guest-agent/ovirt-shutdown-wrappers.h *,\
+    /usr/share/ovirt-guest-agent/ovirt-shutdown-wrappers.sh *,\
     /usr/share/ovirt-guest-agent/ovirt-locksession-wrapper.sh
 
 ovirtagent ALL= NOPASSWD: OVIRTAGENT_SCRIPTS
